### ISSUE TITLE
Timeout larger than 3600 seconds not possible

### ISSUE THREAD

**Issue by FugiOnline** [Jason-Morcos/homebridge-magic-occupancy#5]
Hi,

Unable to define a delay time of larger than 3600 seconds. Is this a limit imposed by Homekit?

**Comment by Jason-Morcos** [comment]
Unable to define as in the homebridge config x UI is blocking you from doing so, or you can't at all? Does it work if you manually update the value in the confit directly? Times longer than 3600 _should_ be possible AFAIK

**Comment by FugiOnline** [comment]
My config is the following:

```json
{
    "name": "Rain Last 24H",
    "stayOccupiedDelay": 86400,
    "persistBetweenReboots": true,
    "startOnReboot": false,
    "triggerSwitchesNames": "Trigger Rain Last 24H",
    "createMasterShutoff": false,
    "accessory": "MagicOccupancy"
}
```

However when I look in Home+ (Version 5) I see the following:
```
Post-Activity Timeout Delay   3.600 s
```

It also seems to switch off the occupancy after an hour in accordance with the 3600 seconds.

**Comment by Jason-Morcos** [comment]
Actually I do see a coded max value of 3600. How strange. I forked this package off of a previous existing package (and improved upon it) and that value appears to be carried forward from the previous package. 
In the upcoming 3.0, I'll remove the limit! Good catch

**Comment by Jason-Morcos** [comment]
Should be available in 3.0.0 beta 14, but beware that upgrading to 3.0 will reset all plugin configs
